documentation/book/cluster-operator.adoc

```diff
@@ -1060,111 +1060,121 @@ They will not be added to the Docker image from a previous build.
 To add multiple plugins to the deployment, they all have to be added within the same build.
 
 
-=== Provisioning Role-Based Access Control (RBAC) for the operator
+=== Provisioning Role-Based Access Control (RBAC) for the Cluster Operator
+
+For the operator to function correctly it needs permission within the {ProductPlatformName} cluster to interact with the resources it manages and interacts with. 
+These include the desired resources, such as  `Kafka`, `Kafka Connect` and so on, as well as the managed resources, such as ConfigMaps, Pods, Deployments, StatefulSets, Services and so on.
+Such permission is described in terms of {ProductPlatformName} role-based access control (RBAC) resources:
+
+* ServiceAccount,
+* Role and ClusterRole,
+* RoleBinding and ClusterRoleBinding.
+
+As well as running under its own ServiceAccount with ClusterRoleBindings, the Cluster Operator itself manages some RBAC resources.
```

Comment:
REPLACE:

As well as running under its own ServiceAccount with ClusterRoleBindings, the Cluster Operator itself manages some RBAC resources. This is because some of the components that are deployed need access to {ProductPlatformName} resources. Furthermore, {ProductPlatformName} includes privilege escalation protections which prevent components operating under one ServiceAccount from granting other ServiceAccounts privileges they do not have. This means that for the Cluster Operator to be able to create the ClusterRoleBindings and RoleBindings needed by resources it manages it needs those same privileges itself.


WITH


Due to the fact that {ProductPlatformName} includes privilege escalation protections that prevent components operating under one ServiceAccount from granting other ServiceAccounts privileges they do not have, the Cluster Operator manages some RBAC resources in addition to running under its own ServiceAccount with ClusterRoleBindings.

Reply:
That's not saying quite the same thing (or at least, not what I wanted to convey). The point I was trying to make was that because the CO creates (Cluster)RoleBindings for components it manages it needs those privileges itself, even though it doesn't actually use them directly. I think this is worth calling out because it helps explain (to people unfamiliar with how Kubernetes/OpenShift RBAC works) why the CO needs certain privileges which it otherwise doesn't seem to use. Maybe this could be a `NOTE` though?

Reply:
Let me consult with some of the other writers on this.

Reply:
@tombentley, does this accurately convey what you mean?: 
In addition to running under its own ServiceAccount with ClusterRoleBindings, the Cluster Operator manages some RBAC resources because some of the components need access to {ProductPlatformName} resources. 

{ProductPlatformName} also includes privilege escalation protections that prevent components operating under one ServiceAccount from granting other ServiceAccounts privileges that the granting Service Account does not have. Because the Cluster Operator must be able to create the ClusterRoleBindings and RoleBindings needed by resources it manages, the Cluster Operator must also have those same privileges.

Reply:
Yes, that's better. Thanks!